segfault when rendering "second" video file

Ubuntu 16.04
Kernel 4.8.1
OpenCV 3.1
(HW: Intel 5930k, Nvidia 1080 GTX Driver 370.28)

**reproduce:**
1. start and create NEW project.
2. Edit the vido
3. Render it (no matter if video or image)
4. delete files from work dir (because they dont't get deleted automaticaly)
5. create NEW project
6. edit
7. render
before it starts to render, its crashing.

**cmake log:**
cmake -DENABLE_TESTS=FALSE -DCMAKE_INSTALL_PREFIX=/home/chris/slomobmp -Wno-dev ../src
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using qt5
Qt libraries found at : Qt5::Gui /  
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Checking for module 'libavformat'
--   Found libavformat, version 56.40.101
-- Checking for module 'libavcodec'
--   Found libavcodec, version 56.60.100
-- Checking for module 'libavutil'
--   Found libavutil, version 54.31.100
-- Checking for module 'libswscale'
--   Found libswscale, version 3.1.101
-- checking: avformat.h
--           /usr/include/x86_64-linux-gnu/libavformat
-- checking: avcodec.h
--           /usr/include/x86_64-linux-gnu/libavcodec
-- checking: avutil.h
--           /usr/include/x86_64-linux-gnu/libavutil
-- checking: swscale.h
--           /usr/include/x86_64-linux-gnu/libswscale
-- OpenCV library status:
--  version: 3.1.0
--  libraries: opencv_calib3d;opencv_core;opencv_features2d;opencv_flann;opencv_highgui;opencv_imgcodecs;opencv_imgproc;opencv_ml;opencv_objdetect;opencv_photo;opencv_shape;opencv_stitching;opencv_superres;opencv_video;opencv_videoio;opencv_videostab
--  include path: /usr/local/include;/usr/local/include/opencv
-- FFMPEG libraries are at /usr/lib/x86_64-linux-gnu/libavformat.so;/usr/lib/x86_64-linux-gnu/libavcodec.so;/usr/lib/x86_64-linux-gnu/libavutil.so;/usr/lib/x86_64-linux-gnu/libswscale.so
-- slowmoInfo Bundle will be :  =>  slowmoVideo 
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libGL.so  
-- Found GLUT: /usr/lib/x86_64-linux-gnu/libglut.so  
-- Found JPEG: /usr/lib/x86_64-linux-gnu/libjpeg.so  
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8") 
-- Found PNG: /usr/lib/x86_64-linux-gnu/libpng.so (found version "1.2.54") 

======================V3D============================
* (info) Installation prefix: /home/chris/slomobmp.
         (Can be adjusted with -DCMAKE_INSTALL_PREFIX=your_path.  Default: .)
* (info) Shaders will be included in the binary: YES
         (Can be disabled with the cmake flag -DDISABLE_INCLUDE_SOURCE)
* (ok) OpenGL found in /usr/include: /usr/lib/x86_64-linux-gnu/libGLU.so;/usr/lib/x86_64-linux-gnu/libGL.so
* (ok) GLUT found in /usr/include: /usr/lib/x86_64-linux-gnu/libglut.so;GLUT_Xmu_LIBRARY-NOTFOUND;/usr/lib/x86_64-linux-gnu/libXi.so
* (ok) GLEW found at /usr/include/GL
* (ok) JPEG libraries found at /usr/include: /usr/lib/x86_64-linux-gnu/libjpeg.so
* (ok) PNG libraries found at /usr/include;/usr/include
* V3D will be built:                        ---YES---
==================slowmoVideo========================
* (info) slowmoVideo installation goes to /home/chris/slomobmp.
         (Can be adjusted with -DCMAKE_INSTALL_PREFIX=your_path. Default is .)
* (ok) ffmpeg found at 
* (ok) OpenCV 3.1.0 found at /usr/local/include;/usr/local/include/opencv.
* slowmoVideo will be built:                ---YES---
=======================END===========================

-- Configuring done
-- Generating done
-- Build files have been written to: /home/chris/Downloads/slowmoVideo/build


**make log:**
Scanning dependencies of target sVinfo
Scanning dependencies of target V3D
[  0%] Generating moc_imageDisplay.cpp
[  1%] Generating moc_ffmpeg_writer.cpp
Scanning dependencies of target sVflow
[  2%] Generating moc_combinedShortcuts.cpp
Scanning dependencies of target sV
[  3%] Building CXX object slowmoVideo/lib/CMakeFiles/sVflow.dir/kernel_sV.cpp.o
[  3%] Building CXX object CMakeFiles/V3D.dir/V3D/GL/glsl_shaders.cpp.o
Scanning dependencies of target sVencode
[  4%] Building CXX object slowmoVideo/lib/CMakeFiles/sVflow.dir/flowField_sV.cpp.o
[  4%] Building CXX object slowmoVideo/lib/CMakeFiles/sVflow.dir/flowTools_sV.cpp.o
[  4%] Building CXX object slowmoVideo/lib/CMakeFiles/sVflow.dir/flowRW_sV.cpp.o
[  5%] Building CXX object CMakeFiles/V3D.dir/V3D/GL/v3d_gpubase.cpp.o
[  5%] Building CXX object CMakeFiles/V3D.dir/V3D/GL/v3d_gpuflow.cpp.o
[  6%] Building CXX object CMakeFiles/V3D.dir/V3D/GL/v3d_gpucolorflow.cpp.o
Scanning dependencies of target sVgui
[  7%] Building CXX object slowmoVideo/lib/CMakeFiles/sV.dir/defs_sV.cpp.o
[  7%] Building CXX object slowmoVideo/lib/CMakeFiles/sVinfo.dir/videoInfo_sV.cpp.o
[  7%] Building CXX object slowmoVideo/lib/CMakeFiles/sVencode.dir/ffmpeg_writer.cpp.o
[  7%] Building CXX object slowmoVideo/libgui/CMakeFiles/sVgui.dir/imageDisplay.cpp.o
[  8%] Building CXX object slowmoVideo/lib/CMakeFiles/sV.dir/vector_sV.cpp.o
[  9%] Building CXX object slowmoVideo/lib/CMakeFiles/sVinfo.dir/avconvInfo_sV.cpp.o
[  9%] Building CXX object CMakeFiles/V3D.dir/V3D/GL/v3d_gpupyramid.cpp.o
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/lib/ffmpeg_writer.cpp:70:0: warning: ignoring #pragma mark  [-Wunknown-pragmas]
 #pragma mark -
 ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/lib/ffmpeg_writer.cpp:133:0: warning: ignoring #pragma mark  [-Wunknown-pragmas]
 #pragma mark -
 ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/lib/ffmpeg_writer.cpp:134:0: warning: ignoring #pragma mark C [-Wunknown-pragmas]
 #pragma mark C bridge
 ^
[ 10%] Building CXX object CMakeFiles/V3D.dir/V3D/Base/v3d_image.cpp.o
[ 11%] Linking CXX static library libsVflow.a
[ 11%] Built target sVflow
Scanning dependencies of target sVvis
[ 12%] Building CXX object slowmoVideo/lib/CMakeFiles/sVvis.dir/flowVisualization_sV.cpp.o
/home/chris/Downloads/slowmoVideo/src/V3D/GL/v3d_gpuflow.cpp:17:4: warning: ‘void {anonymous}::upsampleDisparities(unsigned int, unsigned int, float, V3D_GPU::RTT_Buffer&, V3D_GPU::RTT_Buffer&)’ defined but not used [-Wunused-function]
    upsampleDisparities(unsigned uvSrcTex, unsigned pSrcTex, float pScale,
    ^
/home/chris/Downloads/slowmoVideo/src/V3D/GL/v3d_gpuflow.cpp:67:4: warning: ‘void {anonymous}::upsampleBuffer(unsigned int, float, V3D_GPU::FrameBufferObject&)’ defined but not used [-Wunused-function]
    upsampleBuffer(unsigned srcTex, float scale, FrameBufferObject& dstFbo)
    ^
/home/chris/Downloads/slowmoVideo/src/V3D/GL/v3d_gpuflow.cpp:103:4: warning: ‘void {anonymous}::upsampleBuffers(unsigned int, unsigned int, float, float, V3D_GPU::FrameBufferObject&)’ defined but not used [-Wunused-function]
    upsampleBuffers(unsigned src1Tex, unsigned src2Tex, float scale1, float scale2,
    ^
/home/chris/Downloads/slowmoVideo/src/V3D/GL/v3d_gpucolorflow.cpp:74:4: warning: ‘void {anonymous}::upsampleDisparities(unsigned int, unsigned int, unsigned int, float, V3D_GPU::RTT_Buffer&, V3D_GPU::RTT_Buffer&, V3D_GPU::RTT_Buffer&)’ defined but not used [-Wunused-function]
    upsampleDisparities(unsigned uvSrcTex, unsigned pSrcTex, unsigned qSrcTex, float pScale,
    ^
[ 13%] Building CXX object slowmoVideo/libgui/CMakeFiles/sVgui.dir/combinedShortcuts.cpp.o
[ 14%] Building CXX object slowmoVideo/lib/CMakeFiles/sVencode.dir/video_enc.cpp.o
[ 14%] Building CXX object slowmoVideo/libgui/CMakeFiles/sVgui.dir/moc_imageDisplay.cpp.o
[ 14%] Building CXX object slowmoVideo/lib/CMakeFiles/sV.dir/shutter_sV.cpp.o
[ 14%] Linking CXX static library libV3D.a
[ 14%] Built target V3D
Scanning dependencies of target slowmoFlowBuilder
[ 15%] Building CXX object slowmoVideo/lib/CMakeFiles/sV.dir/intMatrix_sV.cpp.o
[ 16%] Building CXX object V3D/Apps/GL/CMakeFiles/slowmoFlowBuilder.dir/flowBuilder.cpp.o
[ 16%] Building CXX object slowmoVideo/lib/CMakeFiles/sV.dir/interpolate_sV.cpp.o
[ 17%] Building CXX object slowmoVideo/lib/CMakeFiles/sV.dir/bezierTools_sV.cpp.o
[ 17%] Building CXX object V3D/Apps/GL/CMakeFiles/slowmoFlowBuilder.dir/__/__/__/slowmoVideo/lib/flowField_sV.cpp.o
[ 17%] Linking CXX static library libsVvis.a
[ 18%] Building CXX object V3D/Apps/GL/CMakeFiles/slowmoFlowBuilder.dir/__/__/__/slowmoVideo/lib/flowRW_sV.cpp.o
[ 18%] Built target sVvis
Scanning dependencies of target slowmoVisualizeFlow
[ 18%] Building CXX object slowmoVideo/lib/CMakeFiles/sV.dir/sourceField_sV.cpp.o
[ 19%] Building CXX object slowmoVideo/visualizeFlow/CMakeFiles/slowmoVisualizeFlow.dir/visualizeFlow.cpp.o
[ 19%] Building CXX object slowmoVideo/lib/CMakeFiles/sVencode.dir/moc_ffmpeg_writer.cpp.o
[ 19%] Linking CXX executable slowmoFlowBuilder
[ 20%] Building CXX object slowmoVideo/libgui/CMakeFiles/sVgui.dir/moc_combinedShortcuts.cpp.o
[ 20%] Built target slowmoFlowBuilder
[ 20%] Linking CXX static library libsVinfo.a
[ 20%] Built target sVinfo
[ 21%] Linking CXX static library libsV.a
[ 22%] Linking CXX executable slowmoVisualizeFlow
[ 22%] Built target sV
[ 23%] Linking CXX static library libsVgui.a
Scanning dependencies of target slowmoInfo
Scanning dependencies of target slowmoVideoInfo
Scanning dependencies of target slowmoInterpolate
[ 24%] Building CXX object slowmoVideo/slowmoCLI/CMakeFiles/slowmoVideoInfo.dir/videoInfo.cpp.o
[ 25%] Building CXX object slowmoVideo/slowmoInfo/CMakeFiles/slowmoInfo.dir/slowmoInfo.cpp.o
[ 26%] Building CXX object slowmoVideo/slowmoCLI/CMakeFiles/slowmoInterpolate.dir/main.cpp.o
[ 27%] Linking CXX static library libsVencode.a
[ 27%] Linking CXX executable slowmoVideoInfo
[ 27%] Built target sVgui
[ 28%] Generating moc_mainwindow.cpp
[ 29%] Generating ui_shortcutListDialog.h
[ 30%] Generating moc_flowEditCanvas.cpp
[ 31%] Generating ui_mainwindow.h
[ 31%] Generating moc_shortcutListDialog.cpp
[ 31%] Generating ui_flowEditCanvas.h
Scanning dependencies of target slowmoFlowEdit
[ 31%] Built target sVencode
[ 31%] Generating moc_videoFrameSource_sV.cpp
[ 32%] Generating moc_project_sV.cpp
[ 32%] Generating moc_work_flow.cpp
[ 33%] Generating moc_emptyFrameSource_sV.cpp
[ 33%] Generating moc_renderTask_sV.cpp
[ 34%] Generating moc_abstractFrameSource_sV.cpp
[ 35%] Generating moc_imagesFrameSource_sV.cpp
Scanning dependencies of target sVproj
[ 36%] Building CXX object slowmoVideo/slowmoFlowEdit/CMakeFiles/slowmoFlowEdit.dir/mainwindow.cpp.o
[ 36%] Building CXX object slowmoVideo/slowmoFlowEdit/CMakeFiles/slowmoFlowEdit.dir/main.cpp.o
[ 36%] Building CXX object slowmoVideo/slowmoFlowEdit/CMakeFiles/slowmoFlowEdit.dir/moc_flowEditCanvas.cpp.o
[ 37%] Building CXX object slowmoVideo/slowmoFlowEdit/CMakeFiles/slowmoFlowEdit.dir/shortcutListDialog.cpp.o
[ 37%] Building CXX object slowmoVideo/slowmoFlowEdit/CMakeFiles/slowmoFlowEdit.dir/flowEditCanvas.cpp.o
[ 38%] Building CXX object slowmoVideo/slowmoFlowEdit/CMakeFiles/slowmoFlowEdit.dir/moc_shortcutListDialog.cpp.o
[ 39%] Building CXX object slowmoVideo/slowmoFlowEdit/CMakeFiles/slowmoFlowEdit.dir/moc_mainwindow.cpp.o
[ 39%] Built target slowmoVisualizeFlow
[ 40%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/project_sV.cpp.o
[ 40%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/projectPreferences_sV.cpp.o
[ 40%] Built target slowmoVideoInfo
[ 41%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/renderPreferences_sV.cpp.o
[ 41%] Linking CXX executable slowmoInfo
[ 42%] Linking CXX executable slowmoInterpolate
[ 42%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/tag_sV.cpp.o
[ 43%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/node_sV.cpp.o
[ 44%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/nodeList_sV.cpp.o
[ 44%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/segment_sV.cpp.o
[ 44%] Built target slowmoInterpolate
[ 45%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/segmentList_sV.cpp.o
[ 45%] Built target slowmoInfo
[ 45%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/nodeHandle_sV.cpp.o
[ 46%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/renderTask_sV.cpp.o
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/renderTask_sV.cpp:77:0: warning: ignoring #pragma mark  [-Wunknown-pragmas]
 #pragma mark - 
 ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/renderTask_sV.cpp:78:0: warning: ignoring #pragma mark progress [-Wunknown-pragmas]
 #pragma mark progress dialog
 ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/renderTask_sV.cpp:111:0: warning: ignoring #pragma mark  [-Wunknown-pragmas]
 #pragma mark - 
 ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/renderTask_sV.cpp:112:0: warning: ignoring #pragma mark set [-Wunknown-pragmas]
 #pragma mark set/get task
 ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/renderTask_sV.cpp:160:0: warning: ignoring #pragma mark  [-Wunknown-pragmas]
 #pragma mark - 
 ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/renderTask_sV.cpp:161:0: warning: ignoring #pragma mark rendering [-Wunknown-pragmas]
 #pragma mark rendering
 ^
[ 46%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/xmlProjectRW_sV.cpp.o
[ 47%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/abstractFrameSource_sV.cpp.o
[ 48%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/imagesFrameSource_sV.cpp.o
[ 48%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/videoFrameSource_sV.cpp.o
[ 49%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/emptyFrameSource_sV.cpp.o
[ 49%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/abstractRenderTarget_sV.cpp.o
[ 50%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/imagesRenderTarget_sV.cpp.o
[ 50%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/videoRenderTarget_sV.cpp.o
[ 51%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/abstractFlowSource_sV.cpp.o
[ 52%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/flowSourceOpenCV_sV.cpp.o
[ 52%] Linking CXX executable slowmoFlowEdit
[ 52%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/flowSourceV3D_sV.cpp.o
[ 53%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/interpolator_sV.cpp.o
[ 53%] Built target slowmoFlowEdit
[ 53%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/shutterFunction_sV.cpp.o
[ 54%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/shutterFunctionList_sV.cpp.o
[ 54%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/motionBlur_sV.cpp.o
[ 55%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/exportVideoRenderTarget.cpp.o
[ 55%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/work_flow.cpp.o
[ 56%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/moc_project_sV.cpp.o
[ 57%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/moc_renderTask_sV.cpp.o
[ 57%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/moc_abstractFrameSource_sV.cpp.o
[ 58%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/moc_imagesFrameSource_sV.cpp.o
[ 58%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/moc_videoFrameSource_sV.cpp.o
[ 59%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/moc_emptyFrameSource_sV.cpp.o
[ 59%] Building CXX object slowmoVideo/project/CMakeFiles/sVproj.dir/moc_work_flow.cpp.o
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp: In member function ‘void WorkerFlow::doWorkFlow()’:
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:96:9: warning: variable ‘startFrame’ set but not used [-Wunused-but-set-variable]
     int startFrame;
         ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:121:15: warning: unused variable ‘levels’ [-Wunused-variable]
     const int levels = 3;
               ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:122:15: warning: unused variable ‘winsize’ [-Wunused-variable]
     const int winsize = 15;
               ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:123:15: warning: unused variable ‘iterations’ [-Wunused-variable]
     const int iterations = 8;
               ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:125:18: warning: unused variable ‘polySigma’ [-Wunused-variable]
     const double polySigma = 1.2;
                  ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:126:18: warning: unused variable ‘pyrScale’ [-Wunused-variable]
     const double pyrScale = 0.5; // classical pyr
                  ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:127:15: warning: unused variable ‘polyN’ [-Wunused-variable]
     const int polyN = 5;
               ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/project/work_flow.cpp:128:9: warning: unused variable ‘flags’ [-Wunused-variable]
     int flags = 0;
         ^
[ 60%] Linking CXX static library libsVproj.a
[ 60%] Built target sVproj
[ 61%] Generating qrc_resources.cpp
[ 61%] Generating moc_renderPreview.cpp
[ 61%] Generating moc_slowmoRenderer_sV.cpp
[ 62%] Generating moc_mainwindow.cpp
[ 63%] Generating moc_canvas.cpp
[ 63%] Generating moc_frameMonitor.cpp
[ 64%] Generating dialogues/moc_newProjectDialog.cpp
[ 65%] Generating dialogues/moc_renderingDialog.cpp
[ 66%] Generating dialogues/moc_preferencesDialog.cpp
[ 66%] Generating dialogues/moc_projectPreferencesDialog.cpp
[ 66%] Generating dialogues/moc_progressDialog.cpp
[ 67%] Generating moc_flowEditCanvas.cpp
[ 68%] Generating dialogues/moc_shutterFunctionDialog.cpp
[ 69%] Generating dialogues/moc_flowExaminer.cpp
[ 69%] Generating dialogues/moc_shutterFunctionFrame.cpp
[ 70%] Generating dialogues/moc_aboutDialog.cpp
[ 70%] Generating dialogues/moc_tagAddDialog.cpp
[ 71%] Generating ui_canvas.h
[ 71%] Generating ui_frameMonitor.h
[ 71%] Generating moc_notificator.cpp
Scanning dependencies of target slowmoRenderer
[ 72%] Generating ui_renderPreview.h
[ 73%] Generating ui_mainwindow.h
[ 73%] Generating ui_flowEditCanvas.h
[ 74%] Generating ui_newProjectDialog.h
[ 74%] Generating ui_preferencesDialog.h
[ 75%] Generating ui_projectPreferencesDialog.h
[ 76%] Generating ui_progressDialog.h
[ 77%] Generating ui_shutterFunctionDialog.h
[ 77%] Generating ui_flowExaminer.h
[ 77%] Generating ui_renderingDialog.h
[ 78%] Generating ui_tagAddDialog.h
[ 78%] Generating ui_aboutDialog.h
Scanning dependencies of target slowmoUI
[ 79%] Building CXX object slowmoVideo/slowmoRenderer/CMakeFiles/slowmoRenderer.dir/rendererMain.cpp.o
[ 79%] Building CXX object slowmoVideo/slowmoRenderer/CMakeFiles/slowmoRenderer.dir/slowmoRenderer_sV.cpp.o
[ 80%] Building CXX object slowmoVideo/slowmoRenderer/CMakeFiles/slowmoRenderer.dir/moc_slowmoRenderer_sV.cpp.o
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/slowmoRenderer/slowmoRenderer_sV.cpp:185:6: warning: #warning "using fork version" [-Wcpp]
     #warning "using fork version"
      ^
[ 80%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/main.cpp.o
[ 81%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/renderPreview.cpp.o
[ 81%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/frameMonitor.cpp.o
[ 82%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/mainwindow.cpp.o
[ 83%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/flowEditCanvas.cpp.o
[ 84%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/canvas.cpp.o
[ 84%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/canvasTools.cpp.o
[ 85%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/preferencesDialog.cpp.o
[ 85%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/newProjectDialog.cpp.o
[ 86%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/projectPreferencesDialog.cpp.o
[ 86%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/progressDialog.cpp.o
[ 87%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/renderingDialog.cpp.o
[ 87%] Linking CXX executable slowmoRenderer
[ 87%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/shutterFunctionDialog.cpp.o
[ 88%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/shutterFunctionFrame.cpp.o
[ 88%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/flowExaminer.cpp.o
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/slowmoUI/dialogues/renderingDialog.cpp:222:3: warning: #warning "using QTKit version" [-Wcpp]
  #warning "using QTKit version"
   ^
/home/chris/Downloads/slowmoVideo/src/slowmoVideo/slowmoUI/dialogues/renderingDialog.cpp:226:3: warning: #warning "using fork version" [-Wcpp]
  #warning "using fork version"
   ^
[ 89%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/tagAddDialog.cpp.o
[ 90%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/aboutDialog.cpp.o
[ 90%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/notificator.cpp.o
[ 90%] Built target slowmoRenderer
[ 91%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/moc_mainwindow.cpp.o
[ 91%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/moc_canvas.cpp.o
[ 92%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/moc_frameMonitor.cpp.o
[ 92%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/moc_flowEditCanvas.cpp.o
[ 93%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/moc_renderPreview.cpp.o
[ 93%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_newProjectDialog.cpp.o
[ 94%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_preferencesDialog.cpp.o
[ 95%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_projectPreferencesDialog.cpp.o
[ 95%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_progressDialog.cpp.o
[ 96%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_renderingDialog.cpp.o
[ 96%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_shutterFunctionDialog.cpp.o
[ 97%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_shutterFunctionFrame.cpp.o
[ 97%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_flowExaminer.cpp.o
[ 98%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_tagAddDialog.cpp.o
[ 99%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/dialogues/moc_aboutDialog.cpp.o
[ 99%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/moc_notificator.cpp.o
[100%] Building CXX object slowmoVideo/slowmoUI/CMakeFiles/slowmoUI.dir/qrc_resources.cpp.o
[100%] Linking CXX executable slowmoUI
[100%] Built target slowmoUI


**Core Dump info (gdb)**
....
Extracting frames with  "/usr/bin/ffmpeg" ("-i", "/media/chris/SSD1GB/Video/test/2.mp4", "-f", "image2", "/media/chris/SSD1GB/Video/SloMo/work/frames/orig/frame%05d.bmp")
""
""
cache flow source disable
Move confirmed.
Move confirmed.
Move confirmed.
Move confirmed.
Segment  0 :  "Left node: 0; selected: 0"
QFileInfo::absolutePath: Constructed with empty filename
  target dir  QDir( "/media/chris/SSD1GB/Video/SloMo/work/rendered" , nameFilters = { "*" },  QDir::SortFlags( Name | IgnoreCase ) , QDir::Filters( Dirs|Files|Drives|AllEntries ) )
using classical FFMPEG
rendering worker start in Thread  0x7f8947319740
workflow request
Starting rendering process in Thread  0x7f88e7fff700
Frame snapping in from  0  to  0
ASSERT failure in QList<T>::operator[]: "index out of range", file /opt/qt5/include/QtCore/qlist.h, line 541
Abgebrochen

**Valgrind Debug:**
Extracting frames with  "/usr/bin/ffmpeg" ("-i", "/media/chris/SSD1GB/Video/test/2.mp4", "-f", "image2", "/media/chris/SSD1GB/Video/SloMo/work/frames/orig/frame%05d.bmp")
""
""
cache flow source disable
Move confirmed.
Move confirmed.
Move confirmed.
Segment  0 :  "Left node: 0; selected: 0"
QFileInfo::absolutePath: Constructed with empty filename
  target dir  QDir( "/media/chris/SSD1GB/Video/SloMo/work/rendered" , nameFilters = { "*" },  QDir::SortFlags( Name | IgnoreCase ) , QDir::Filters( Dirs|Files|Drives|AllEntries ) )
using classical FFMPEG
rendering worker start in Thread  0x40a7180
workflow request
Starting rendering process in Thread  0x26e62700
==12304== Thread 7 QThread:
==12304== Invalid read of size 8
==12304==    at 0x4AF082: Project_sV::snapToOutFrame(double, bool, Fps_sV const&, int*) const (project_sV.cpp:368)
==12304==    by 0x4BC272: RenderTask_sV::slotContinueRendering() (renderTask_sV.cpp:180)
==12304==    by 0x4DDBA0: RenderTask_sV::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) (moc_renderTask_sV.cpp:119)
==12304==    by 0x5F166A8: QMetaObject::activate(QObject*, int, int, void**) (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x5D12CD1: ??? (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x82636F9: start_thread (pthread_create.c:333)
==12304==    by 0x7F99B5C: clone (clone.S:109)
==12304==  Address 0x2290ff30 is 8 bytes after a block of size 136 free'd
==12304==    at 0x4C2EDEB: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==12304==    by 0x15DB76BA: ??? (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x15DCEF2F: ??? (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x15DA0FF8: gdk_display_get_event (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x15DCEC91: ??? (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x9A4F1A6: g_main_context_dispatch (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x9A4F3FF: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x9A4F721: g_main_loop_run (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x155CC41F: gtk_dialog_run (in /usr/lib/x86_64-linux-gnu/libgtk-3.so.0.1800.9)
==12304==    by 0x15408A11: ??? (in /opt/qt-5.7.0/plugins/platformthemes/libqgtk3.so)
==12304==    by 0x54091B7: QDialog::exec() (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x49082F: RenderingDialog::slotBrowseVideoFile() (renderingDialog.cpp:487)
==12304==  Block was alloc'd at
==12304==    at 0x4C2DB8F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==12304==    by 0x9A54728: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x9A6B932: g_slice_alloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x9A6BFCD: g_slice_alloc0 (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x15DA47FF: gdk_event_new (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x15DCF5AE: ??? (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x15DCEFEF: ??? (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x15DA0FF8: gdk_display_get_event (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x15DCEC91: ??? (in /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.1800.9)
==12304==    by 0x9A4F1A6: g_main_context_dispatch (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x9A4F3FF: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304==    by 0x9A4F721: g_main_loop_run (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.4800.1)
==12304== 
==12304== Invalid read of size 8
==12304==    at 0x45897E: QListData::size() const (qlist.h:113)
==12304==    by 0x4B8795: QList<Node_sV>::length() const (qlist.h:340)
==12304==    by 0x4B41F0: NodeList_sV::endTime(bool) const (nodeList_sV.cpp:52)
==12304==    by 0x4AF092: Project_sV::snapToOutFrame(double, bool, Fps_sV const&, int*) const (project_sV.cpp:368)
==12304==    by 0x4BC272: RenderTask_sV::slotContinueRendering() (renderTask_sV.cpp:180)
==12304==    by 0x4DDBA0: RenderTask_sV::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) (moc_renderTask_sV.cpp:119)
==12304==    by 0x5F166A8: QMetaObject::activate(QObject*, int, int, void**) (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x5D12CD1: ??? (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x82636F9: start_thread (pthread_create.c:333)
==12304==    by 0x7F99B5C: clone (clone.S:109)
==12304==  Address 0x22914a88 is 24 bytes before a block of size 496 free'd
==12304==    at 0x4C2F24B: operator delete(void*) (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==12304==    by 0x528EE1C: QStyleOption::~QStyleOption() (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x52C5C01: ??? (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x531E068: QComboBox::paintEvent(QPaintEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x52654CF: QWidget::event(QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x531F2A5: QComboBox::event(QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x521E8BB: QApplicationPrivate::notify_helper(QObject*, QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x5226275: QApplication::notify(QObject*, QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x5EEB54F: QCoreApplication::notifyInternal2(QObject*, QEvent*) (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x525E319: QWidgetPrivate::sendPaintEvent(QRegion const&) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x525E96F: QWidgetPrivate::drawWidget(QPaintDevice*, QRegion const&, QPoint const&, int, QPainter*, QWidgetBackingStore*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x525F63B: QWidgetPrivate::paintSiblingsRecursive(QPaintDevice*, QList<QObject*> const&, int, QRegion const&, QPoint const&, int, QPainter*, QWidgetBackingStore*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==  Block was alloc'd at
==12304==    at 0x4C2E0EF: operator new(unsigned long) (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==12304==    by 0x5846B6A: QPalette::detach() (in /opt/qt-5.7.0/lib/libQt5Gui.so.5.7.0)
==12304==    by 0x5846CEB: QPalette::setBrush(QPalette::ColorGroup, QPalette::ColorRole, QBrush const&) (in /opt/qt-5.7.0/lib/libQt5Gui.so.5.7.0)
==12304==    by 0x52B90AF: ??? (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x52C610E: ??? (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x531E068: QComboBox::paintEvent(QPaintEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x52654CF: QWidget::event(QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x531F2A5: QComboBox::event(QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x521E8BB: QApplicationPrivate::notify_helper(QObject*, QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x5226275: QApplication::notify(QObject*, QEvent*) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304==    by 0x5EEB54F: QCoreApplication::notifyInternal2(QObject*, QEvent*) (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x525E319: QWidgetPrivate::sendPaintEvent(QRegion const&) (in /opt/qt-5.7.0/lib/libQt5Widgets.so.5.7.0)
==12304== 
==12304== Invalid read of size 4
==12304==    at 0x458981: QListData::size() const (qlist.h:113)
==12304==    by 0x4B8795: QList<Node_sV>::length() const (qlist.h:340)
==12304==    by 0x4B41F0: NodeList_sV::endTime(bool) const (nodeList_sV.cpp:52)
==12304==    by 0x4AF092: Project_sV::snapToOutFrame(double, bool, Fps_sV const&, int*) const (project_sV.cpp:368)
==12304==    by 0x4BC272: RenderTask_sV::slotContinueRendering() (renderTask_sV.cpp:180)
==12304==    by 0x4DDBA0: RenderTask_sV::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) (moc_renderTask_sV.cpp:119)
==12304==    by 0x5F166A8: QMetaObject::activate(QObject*, int, int, void**) (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x5D12CD1: ??? (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x82636F9: start_thread (pthread_create.c:333)
==12304==    by 0x7F99B5C: clone (clone.S:109)
==12304==  Address 0xc is not stack'd, malloc'd or (recently) free'd
==12304== 
==12304== 
==12304== Process terminating with default action of signal 11 (SIGSEGV): dumping core
==12304==  Access not within mapped region at address 0xC
==12304==    at 0x458981: QListData::size() const (qlist.h:113)
==12304==    by 0x4B8795: QList<Node_sV>::length() const (qlist.h:340)
==12304==    by 0x4B41F0: NodeList_sV::endTime(bool) const (nodeList_sV.cpp:52)
==12304==    by 0x4AF092: Project_sV::snapToOutFrame(double, bool, Fps_sV const&, int*) const (project_sV.cpp:368)
==12304==    by 0x4BC272: RenderTask_sV::slotContinueRendering() (renderTask_sV.cpp:180)
==12304==    by 0x4DDBA0: RenderTask_sV::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) (moc_renderTask_sV.cpp:119)
==12304==    by 0x5F166A8: QMetaObject::activate(QObject*, int, int, void**) (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x5D12CD1: ??? (in /opt/qt-5.7.0/lib/libQt5Core.so.5.7.0)
==12304==    by 0x82636F9: start_thread (pthread_create.c:333)
==12304==    by 0x7F99B5C: clone (clone.S:109)
==12304==  If you believe this happened as a result of a stack
==12304==  overflow in your program's main thread (unlikely but
==12304==  possible), you can try to increase the size of the
==12304==  main thread stack using the --main-stacksize= flag.
==12304==  The main thread stack size used in this run was 8388608.
==12304== 
==12304== HEAP SUMMARY:
==12304==     in use at exit: 21,188,547 bytes in 60,605 blocks
==12304==   total heap usage: 2,609,410 allocs, 2,548,805 frees, 5,572,058,709 bytes allocated
==12304== 
==12304== LEAK SUMMARY:
==12304==    definitely lost: 40,200 bytes in 90 blocks
==12304==    indirectly lost: 49,353 bytes in 1,831 blocks
==12304==      possibly lost: 688,094 bytes in 196 blocks
==12304==    still reachable: 20,155,892 bytes in 56,424 blocks
==12304==                       of which reachable via heuristic:
==12304==                         length64           : 20,336 bytes in 278 blocks
==12304==                         newarray           : 2,568 bytes in 79 blocks
==12304==                         multipleinheritance: 6,912 bytes in 12 blocks
==12304==         suppressed: 0 bytes in 0 blocks
==12304== Rerun with --leak-check=full to see details of leaked memory
==12304== 
==12304== For counts of detected and suppressed errors, rerun with: -v
==12304== Use --track-origins=yes to see where uninitialised values come from
==12304== ERROR SUMMARY: 4273 errors from 6 contexts (suppressed: 0 from 0)
Getötet

--- a/src/slowmoVideo/slowmoUI/mainwindow.cpp
+++ b/src/slowmoVideo/slowmoUI/mainwindow.cpp
@@ -632,7 +632,7 @@ void MainWindow::slotShowRenderDialog()
  
             connect(task, SIGNAL(signalRenderingFinished(QString)), &m_rendererThread, SLOT(quit()));
             // done another way ?!
-            //connect(task, SIGNAL(signalRenderingFinished(QString)), task, SLOT(deleteLat    er()));
+            connect(task, SIGNAL(signalRenderingFinished(QString)), task, SLOT(deleteLater()));
              
             //connect(&m_rendererThread, &QThread::finished, task, &QObject::deleteLater);
             // let's start

